client/src/tokens/objectStyles.js

```diff
@@ -18,6 +18,7 @@ const borderWidth = {
 const boxShadow = {
   DEFAULT: '5px 5px 20px rgba(0, 0, 0, 0.05)',
   md: '5px 5px 30px rgba(0, 0, 0, 0.25)',
+  lg: '5px 5px 30px rgba(0, 0, 0, 0.5)',
```

Comment:
Here we’d be better off doing:

```suggestion
  DEFAULT: '5px 5px 20px rgba(0, 0, 0, 0.05)',
  md: '5px 5px 30px var(--w-color-shadow-md)'
```

…and creating a theme-agnostic "Misc" token (in `colorThemes.js`), something like:

```js
{
  value: 'var(--w-color-black-50)',
  bgUtility: 'w-bg-shadow-md',
  textUtility: 'w-text-shadow-md',
  cssVariable: '--w-color-shadow-md',
}
```

Reply:
committed suggestion